src/Solver/time_stepping/AbstractGhostTimeCluster.cpp

```diff
@@ -116,4 +116,13 @@ void AbstractGhostTimeCluster::printTimeoutMessage(std::chrono::seconds timeSinc
         << "correctionsSinceSync = " << neighbor.ct.stepsSinceLastSync;
   }
 }
+
+ClusterTimes& AbstractGhostTimeCluster::getClusterTimes(){
```

Comment:
Maybe it's better to make this one const, and offer a method to explicitly change the one parameter you want to change?

Reply:
Replaced with a direct method to access the required parameter